pydis_site/apps/api/models/bot/bumped_thread.py

```diff
@@ -0,0 +1,22 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from pydis_site.apps.api.models.mixins import ModelReprMixin
+
+
+class BumpedThread(ModelReprMixin, models.Model):
+    """A list of thread IDs to be bumped."""
+
+    thread_id = models.BigIntegerField(
```

Comment:
Would it make more sense to use a pg array instead, or do frequent membership checks also need to happen?

Reply:
Yea, membership checks will happen quite often to check when a thread is archived if it's in the bump lost.